### ISSUE TITLE
problems with monitoring inventory

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#571]
**Version:** r7 4dc20f7
**Environment:** dmaster

After adding hosts with cv_install_moncon:
Tried to do inventory

```
OMD[monitoring]:~$ cmk -I
Host 'controller': Cannot get data from TCP port 10.141.255.254:6556: [Errno 111] Connection refused
Cannot resolve c001 into IP address.
Cannot resolve c002 into IP address.
Host 'node003': Cannot get data from TCP port 10.141.0.3:6556: [Errno 111] Connection refused
Host 'login.vc-new': Cannot get data from TCP port 172.17.255.254:6556: [Errno 111] Connection refused
Host 'switch': Cannot get data from TCP port 192.168.192.168:6556: [Errno 111] Connection refused
Host 'login.vc-a': Cannot get data from TCP port 172.16.255.254:6556: [Errno 111] Connection refused
Host 'node004': Cannot get data from TCP port 10.141.0.4:6556: [Errno 111] Connection refused
Cannot get information from host 'c001': Cannot contact agent: host 'c001' has no IP address.
Cannot get information from host 'c002': Cannot contact agent: host 'c002' has no IP address.
cpu.loads         2 new checks
cpu.threads       2 new checks
df                4 new checks
diskstat          2 new checks
kernel            6 new checks
kernel.util       2 new checks
lnx_if            2 new checks
logwatch          2 new checks
mem.used          2 new checks
mounts            2 new checks
nfsmounts         6 new checks
ntp.time          1 new checks
tcp_conn_stats    2 new checks
uptime            2 new checks
```

**Comment by hansthen** [comment]
I am not sure what to do with this. I think in our procedures only the compute nodes (so not the nova nodes) get an agent installed. As far as I can see, node001 and node002 are added succesfully, and c001 and c002 do not have an ip address. This may be because of the (re-)partitioning failure.

**Comment by themisa** [comment]
@hansthen they are added in hosts table, but they can't get inventoried. So if you pull up the web interface, there are no hosts, or anything. the cmk -I command is to get inventory from the listed hosts.

**Comment by hansthen** [comment]
Can you attach a screen dump of the web interface?

**Comment by hansthen** [comment]
The containers cannot be monitored from inside the dockerized omd. They are localized to a domain (vc-a, vc-b). This worked by accident on a bare metal host. I propose to remove the containers from monitoring.
 
```
[root@controller(dmaster) ~]# ping c001
PING c001 (172.16.0.1) 56(84) bytes of data.
64 bytes from c001 (172.16.0.1): icmp_seq=1 ttl=64 time=0.229 ms
64 bytes from c001 (172.16.0.1): icmp_seq=2 ttl=64 time=0.224 ms
64 bytes from c001 (172.16.0.1): icmp_seq=3 ttl=64 time=0.221 ms
64 bytes from c001 (172.16.0.1): icmp_seq=4 ttl=64 time=0.222 ms
^C
--- c001 ping statistics ---
4 packets transmitted, 4 received, 0% packet loss, time 2999ms
rtt min/avg/max/mdev = 0.221/0.224/0.229/0.003 ms
[root@controller(dmaster) ~]# vi /etc/resolv.conf 
[root@controller(dmaster) ~]# docker-enter omd
[root@4a092b25b9ac /]# ping c001
ping: unknown host c001
```

**Comment by themisa** [comment]
This fix introduced a new bug #584. Cannot test this until that is fixed.

**Comment by themisa** [comment]
```
ast login: Thu Dec 31 14:33:23 UTC 2015
OMD[monitoring]:~$ cmk -I
cpu.loads         2 new checks
cpu.threads       2 new checks
df                4 new checks
diskstat          2 new checks
kernel            6 new checks
kernel.util       2 new checks
lnx_if            2 new checks
logwatch          2 new checks
mem.used          2 new checks
mounts            2 new checks
nfsmounts         6 new checks
tcp_conn_stats    2 new checks
uptime            2 new checks
```
